src/Core/Renderer/Twig/Function/PrintEntityCollectionAsList.php

```diff
@@ -33,35 +36,41 @@ public static function getOptions(): array
     {
         return [
             'is_safe' => ['html'],
+            'needs_context' => true,
         ];
     }
 
     /**
+     * @param array $context
      * @param RootEntityCollection $rootEntityCollection Processed entity collection
      * @param string $type List tag type (<ul>/<ol>)
      * @param bool $skipDescription Don't print description of this entities
      * @param bool $useFullName Use the full name of the entity in the list
      * @return string
-     * @throws InvalidConfigurationParameterException
      */
     public function __invoke(
+        array $context,
         RootEntityCollection $rootEntityCollection,
         string $type = 'ul',
         bool $skipDescription = false,
         bool $useFullName = false,
     ): string {
-        $result = "<{$type}>";
+        $result = '';
+        $prefix = '1. ';
```

Comment:
This is the issue with the ordered lists, need to increase this as it goes through entities.

Reply:
This is not an mistake, but the correct syntax for a numbered list :)

Reply:
https://www.markdownguide.org/basic-syntax/#ordered-lists wow TIL, markdowns a funny language! 😂